python-api/src/main/python/livy/client.py

```diff
@@ -159,8 +157,7 @@ def add_file(self, file_uri):
 
         Parameters
         ----------
-        file_uri : pathlib.PurePath or pathlib2.PurePath, string in the format
-            of file uri
+        file_path : string
```

Comment:
No, this is not a path, this is a URI, just provided in string form. And unlike what the comment says, it's not a "local" file.

Local files are only for the upload* methods.